### PR TITLE
Add akashnet-2-upgrade-1 software upgrade guide

### PR DESCRIPTION
**Pull request by anilCSE**

### CODE REVIEW

File: guides/akashnet-2-software-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+**Option-2: Build from source**
+
+   - Requires go1.16+
+   ```sh
+   git clone https://github.com/ovrclk/akash
+   cd akash
+   git fetch
+   git checkout v0.12.0
+   make install
```
Comment (boz) [inline guides/akashnet-2-software-upgrade.md:33]: people will insist on using

```sh
MAINNET=true make install
```

File: guides/akashnet-2-software-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Akashnet-2 Software upgrade procedure
+
+## What validators need to do before the upgrade time?
+
+- Nothing, just sit back and relax. The `software-upgrade` proposal will force the binary (current version, `akash-v0.10.x`) to halt at specified height, i.e., 455200
+
+## What validators need to do at the time of upgrade (i.e., at block height `455200`)
+### 1. Cross check your consensus height:
+```sh
+akash status 2>&1 | jq -r '.SyncInfo.latest_block_height'
```
Comment (boz) [inline guides/akashnet-2-software-upgrade.md:10]: The process won't exit?

**Review by boz — COMMENTED** [review]
What happens if the new version fails?  Will starting the old version continue to stay halted?

**Comment by anilCSE** [comment]
> What happens if the new version fails? Will starting the old version continue to stay halted?

Yes, will include those details too

File: guides/akashnet-2-software-upgrade.md (hunk 2 above)
Comment (anilCSE) [reply guides/akashnet-2-software-upgrade.md:10]: Since many are using systemctl, it will restart the node again.

**Review by boz — APPROVED** [review]
🚀

**Review by boz — APPROVED** [review]
🚀

**Pull request merged**